### ISSUE TITLE
Server should initiate a clientside action

### ISSUE THREAD

**Issue by chenblair** [MobileCommerceLab/basic_ionic_channels_chat#6]
As a proof of concept, from the django socket server, a server actions should be made to initiate some simple change on the clientside.

Tentatively, this will be a message clear, where all messages sent so far on the server will be deleted, and all messages should be deleted on any open apps as well.

**Comment by drspangle** [comment]
@chenblair Good. 

Also, you should x-post this to the repo for the serverside code. It would be worth putting a completion checkbox on both that tracks the pieces of the feature that need implemented (concretely, 1) API endpoint, 2) clientside function, 3) serverside model event hook function). This way, if things get complicated, you can implement each piece in a separate PR. It's also fine to do this all in one PR (obviously one in each repo) so long as it's manageable in size.

**Comment by chenblair** [comment]
@drspangle Currently the server-side code is only on my local machine; should I make a separate repo for that?

**Comment by drspangle** [comment]
@chenblair Yeah, absolutely. As per the software requirements you signed yesterday, *all* artifacts must be in version control. We can't risk losing something because it only exists in local storage on someone's machine, or on a server somewhere. The same standard for documentation, good version control practices, etc. apply. It's an extremely important habit to get into.
